src/main/java/model/input/file/DataFile.java

```diff
@@ -311,32 +257,84 @@ public void createMetaDataValue(String name, String type) {
 		try {
 			String fileName = this.getFile().getName();
 			String metaValue = fileName.substring(0, fileName.lastIndexOf("."));
-
-			this.metaDataValue = parseDataValue(metaValue, type);
+			Class<? extends DataValue> typeClass = DataFile.getColumnType(type);
+			this.metaDataValue = parseSimpleDataValue(metaValue, typeClass);
```

Comment:
Simpledatavalue is inderdaad alleen maar string of getallen. Deze kunnen in alle gevallen (txt of excel) naar een waarde geparst worden. Maar dates zijn afhankelijk van de type file waarin ze voorkomen. In een textfile moeten ze worden geparst met een format en in een excelfile kan je een datecell door POI een date laten returnen